.nycrc

```diff
@@ -7,7 +7,8 @@
 		"src/cli/utils/profile.ts",
 		"src/linter/xmlTemplate/lib/JSTokenizer.js",
 		"src/linter/binding/lib/**/*.js",
-		"**/*.d.ts"
+		"**/*.d.ts",
+		"src/autofix/**"
```

Comment:
This is needed temporary as currently we haven't implemented all the autofix functionality and it will be done in the next BLIs